engine/multisig/src/crypto/key_id.rs

```diff
@@ -42,6 +43,12 @@ impl CanonicalEncoding for cf_chains::sol::SolAddress {
 	}
 }
 
+impl CanonicalEncoding for VerifyingKey {
```

Comment:
Nit: I would fully qualify the type here (`ed25519_dalek::VerifyingKey`) since this file isn't specific to ed25519, and bringing `VerifyingKey` into its scope could lead to confusion.